core/templates/dev/head/pages/exploration_player/ExtractImageFilenamesFromStateService.js

```diff
@@ -223,10 +216,9 @@ oppia.factory('ExtractImageFilenamesFromStateService', [
       // directly stored in the customizationArgs.imageAndRegion.value
       // .imagePath
       if (state.interaction.id === INTERACTION_TYPE_IMAGE_CLICK_INPUT) {
-        var filepathObject = (
+        var filename = (
```

Comment:
Seems like this should be "filepath" because it reads off of "imagePath" 

In general we should differentiate between "filepath" (the whole thing) and "filename" (just the last part). (Accordingly, is filenamesInState correct or should it be filepathsInState?)

Reply:
We have the attribute of filepath in oppia-noninteractive-image tag. But it actually contains the filename only, so I think that is the confusion here. 
According to me its good to go with the filename here, since in the end what we are retrieving is the filenames in the exploration.

Reply:
Oh I see. OK I think it's fine to go with filename then, but please add a comment to explain in the code what you've just explained above. Otherwise it's confusing to the reader.

I was thinking about filing a bug that updates filepath to filename in the RTE component, but you're going to change this completely anyway right? If so then doing this (updating "filepath" to "filename") would probably not be useful and I won't file it.

Reply:
Okay, will add it. Yeah, I think it'll be better to do that later on. Thanks.

Reply:
Done!

Reply:
Add the explanation for filename directly before this line too. Since this is different method and reader might not read the explanation.